app/controllers/rooms_controller.rb

```diff
@@ -13,9 +13,18 @@ class RoomsController < ApplicationController
 
   def show
     room = Room.find(params[:id])
-    ActionCable.server.broadcast 'songs', songs: room.songs, type: 'songs'
+
+    ActionCable.server.broadcast('queue', songs_for(room))
 
     raise JsonapiCompliable::Errors::RecordNotFound unless room
     render_jsonapi(room, scope: false)
   end
+
+  private
+
+  def songs_for(room)
+    JSONAPI::Serializable::Renderer
```

Comment:
Is this a JSON suite thing?

Reply:
Yup, it's what json suite uses internally when we call like `render_jsonapi`